src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/courtsentencing/CourtSentencingMigrationService.kt

```diff
@@ -90,31 +89,6 @@ class CourtSentencingMigrationService(
       }
   }
 
-  // temp until DPS handles linked cases
```

Comment:
main change, no longer ignore these case but use the DPS rules to link the cases